assets/app/scripts/extensions/javaLink.js

```diff
@@ -40,25 +40,27 @@ angular.module('openshiftConsoleExtensions', ['openshiftConsole'])
       if (!jolokiaPort) {
         return;
       }
-      var pod = $scope.$eval('pod');
+      var pod = $scope.$eval('podTemplate');
```

Comment:
you can't count on the scope 'podTemplate' object being an actual pod... in most cases, they are pod templates, and proxying to their id as a pod won't work